src/simtools/model/model_parameter.py

```diff
@@ -351,12 +351,8 @@ def _load_simulation_software_parameter(self):
                         simulation_software=simulation_software,
                     )
                 )
-            except ValueError as exc:
```

Comment:
Should we consider printing out the warning just once, when the first failure occurs, instead of completely ignoring it?

It could be done with a class variable which will be used as a flag to print or not the warning.

Reply:
I am against it. It happens each time when this function is called, so this is expected behavior.

Reply:
OK!